app/presenters/summary/sections/codefendants.rb

```diff
@@ -42,7 +42,7 @@ def name_and_conflict(codefendant)
         if YesNoAnswer.new(codefendant.conflict_of_interest).yes?
```

Comment:
We could now refactor this so there is no need for IF-ELSE
We can pass the value of `codefendant.conflict_of_interest` to the I18n

Like:
```[codefendant.full_name, I18n.t(codefendant.conflict_of_interest, scope: 'summary.questions.conflict_of_interest_html')].join```

Reply:
Yep, of course. Makes sense. Thank you 👍